src/Plugin.Maui.Audio/AudioPlayer.shared.cs

```diff
@@ -2,7 +2,8 @@
 
 public partial class AudioPlayer : IAudioPlayer
 {
-    public event EventHandler? PlaybackEnded;
+#pragma warning disable CS0067
+	public event EventHandler? PlaybackEnded;
```

Comment:
Do we need to re-enable the warning afterwards? e.g.

```suggestion
#pragma warning disable CS0067
	public event EventHandler? PlaybackEnded;
#pragma warning enable CS0067
```

Reply:
Good catch - 'restore' instead of enable I think

Reply:
Oh yes it is never the term I expect